### PR TITLE
Changed library from doc_opt to command parser

### PR DESCRIPTION
**Pull request by liealex**

### CODE REVIEW

File: examples/point_cloud/main.cpp [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
-#include "docopt.h"
+#include <command_parser.h>
+#include <aditof/log.h>
```
Comment (SeptimiuVana) [inline examples/point_cloud/main.cpp:3]: can you link it to glog as well? we use this header only when glog dependency is set to off.

Comment (liealex) [reply examples/point_cloud/main.cpp:6]: Solved

**Review by dNechita — APPROVED** [review]

**Review by dNechita — CHANGES_REQUESTED** [review]
Can you point this to branch: dev-4.4.0 instead of master?

**Comment by dNechita** [comment]
You need to rebase your branch onto dev-4.4.0. This PR includes commits that do not belong here.

**Review by dNechita — APPROVED** [review]

**Pull request merged**